config.toml

```diff
@@ -32,9 +32,9 @@ googleAnalytics = ""
     # - send a dummy email yourself to confirm your account
     # - click the confirm link in the email from www.formspree.io
     # - you're done. Happy mailing!
-	email = "your@email.com"
+	email = "ryan.walls@3dsim.com"
```

Comment:
What email is this? we have several corporate emails we could use or create a new one if this will be accessible on the site